core/src/main/resources/template/simple/optgroup.ftl

```diff
@@ -35,10 +35,37 @@
 	<#assign tmpKey=stack.findValue(optGroupInternalListUiBean.parameters.listKey) />
 	<#assign tmpValue=stack.findValue(optGroupInternalListUiBean.parameters.listValue) />
 	<#assign tmpKeyStr = tmpKey.toString() />
+	<#assign optGroupItemCssClass = ''/>
+	<#if optGroupInternalListUiBean.parameters.listCssClass??>
+		<#if stack.findString(optGroupInternalListUiBean.parameters.listCssClass)??>
+			<#assign optGroupItemCssClass= stack.findString(optGroupInternalListUiBean.parameters.listCssClass)/>
```

Comment:
Such repeatit evaluation can be expensive, why not to do it once and assign the value? In case of missing expression, a `null` will be returned which should work with `?has_content`.

Reply:
Copy paste from select template actually. I've seen this and thought about improving it later in separate commit. :)
Freemarker doesn't allow to assign null / undefined, so the default value operator should be used.

Reply:
👏